### ISSUE TITLE
bedpesort doesn't work correctly with filename provided on command line

### ISSUE THREAD

**Issue by ernfrid** [hall-lab/svtools#134]
```
$ svtools bedpesort /tmp/dlarson_testing/converted.bedpe
/tmp/dlarson_testing/miniconda/envs/svtools0.2.0/lib/python2.7/site-packages/setuptools-21.2.1-py2.7.egg/pkg_resources/__init__.py:1268: UserWarning: /gscuser/dlarson/.python-eggs is writable by group/others and vulnerable to attack when used with get_resource_filename. Consider a more secure location (set with .set_extraction_path or the PYTHON_EGG_CACHE environment variable).
Running bedpesort with options: /gscuser/dlarson/.python-eggs/svtools-0.2.0-py2.7.egg-tmp/svtools/bin/bedpesort /tmp/dlarson_testing/converted.bedpe
awk: cannot open /tmp/dlarson_testing/converted.bedpe (Value too large for defined data type)
bedpesort exited with code 2
```

**Comment by ernfrid** [comment]
Should also check on vcfsort...

**Comment by ernfrid** [comment]
Fix in #153
